backend/src/main/java/com/google/models/Anomaly.java

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.models;
+
+import com.google.models.MetricValue;
+import com.google.models.Timestamp;
+import java.util.Map;
+
+/** Store anomaly-related data. */
+public final class Anomaly {
+  
+  private final Timestamp timestampDate;
+  private final String metricName;
+  private final String dimensionName;
+  private final String status;
```

Comment:
what is the status of an anomaly?

Reply:
it refers to whether an anomaly is resolved or not. 
i will clarify it with the naming.